include/mbgl/style/expression/interpolate.hpp

```diff
@@ -73,7 +73,7 @@ class InterpolateBase : public Expression {
 
     void eachChild(const std::function<void(const Expression&)>& visit) const override {
         visit(*input);
-        for (const std::pair<const double, const std::unique_ptr<Expression>&>& stop : stops) {
+        for (const std::pair<const double, std::unique_ptr<Expression, std::default_delete<Expression>>>& stop : stops) {
```

Comment:
can we replace this with `auto` as well?

Reply:
/cc @anandthakker @jfirebaugh

Reply:
→ 142b1f1